contracts/src/v0.8/workflow/dev/WorkflowRegistry.sol

```diff
@@ -83,8 +83,8 @@ contract WorkflowRegistry is Ownable2StepMsgSender, ITypeAndVersion {
     bytes32 indexed workflowID, address indexed workflowOwner, uint32 indexed donID, string workflowName
   );
   event WorkflowForceUpdateSecretsRequestedV1(address indexed owner, bytes32 secretsURLHash, string workflowName);
-  event RegistryLockedV1(address indexed lockedBy);
```

Comment:
why removing these?

Reply:
The event is highly infrequent and the index won't help much with anything except additional gas cost

Reply:
And this will always be the contract owner only, right?

Reply:
Now, when I think about it again, maybe this event doesn't even have to emit this information. The owner will always be an MSIG contract, probably MCMS.